emoji/src/main/java/com/vanniktech/emoji/EmojiPopup.java

```diff
@@ -331,8 +342,12 @@ public void dismiss() {
 
   void showAtBottom() {
     isPendingOpen = false;
-    popupWindow.showAtLocation(rootView, Gravity.NO_GRAVITY, 0,
-        Utils.getProperHeight(context) + popupWindowHeight);
+    editText.postDelayed(new Runnable() {
+      @Override public void run() {
+        popupWindow.showAtLocation(rootView, Gravity.NO_GRAVITY, 0,
+            Utils.getProperHeight(context) + popupWindowHeight);
+      }
+    }, delay);
```

Comment:
Is this safe in regard to race conditions?

Reply:
No. Got better options? :)

Reply:
Sorry, no, I can't really follow the flow in `EmojiPopup` anymore :/